src/scripts/main.js

```diff
@@ -2,7 +2,35 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('table');
 
-// write your code here
+for (const person of people) {
+  const row = document.createElement('tr');
+  const column1 = document.createElement('td');
+  const column2 = document.createElement('td');
+  const column3 = document.createElement('td');
```

Comment:
```suggestion
  const born = document.createElement('td');
```
Назови так, чтобы было ясно что в колонках